grommet-leaflet-core/src/MarkerCluster.jsx

```diff
@@ -9,22 +10,43 @@ import 'leaflet.markercluster';
 import ReactDOMServer from 'react-dom/server';
 import { Cluster, Popup } from './markers';
 
-const createMarkerClusterGroup = ({ popup: popupProp, ...rest }, context) => {
+const createMarkerClusterGroup = (
+  { icon: iconProp, popup: popupProp, ...rest },
+  context,
+) => {
+  const theme = React.useContext(ThemeContext);
+
   const markerClusterGroup = new L.MarkerClusterGroup({
     zoomToBoundsOnClick: false,
     iconCreateFunction: cluster => {
-      const popup = cluster.bindPopup(
-        ReactDOMServer.renderToString(<Popup>{popupProp(cluster)}</Popup>),
-      );
+      if (popupProp) {
+        const popup = cluster.bindPopup(
```

Comment:
While we are here, should we support popup for Marker as well?

Reply:
I think that is reasonable, I will add.

Reply:
Done.